tensorrt_llm/_torch/pyexecutor/model_engine.py

```diff
@@ -527,7 +538,9 @@ def no_cuda_graph():
                         logger.info(f"Run autotuning warmup for batch size={1}")
                         self.forward(batch,
                                      new_tensors_device=None,
-                                     resource_manager=resource_manager)
+                                     resource_manager=resource_manager,
+                                     extra_model_inputs=_create_extra_inputs(
+                                         1, num_tokens))
```

Comment:
Why bs = 1 here?

Reply:
This one is doing a warmup with a single request with `num_tokens`. See L532.